api_docs/video_favorites.md

```diff
@@ -4,49 +4,128 @@ title: Zype Developer Portal | Video Favorites
 permalink: /api_docs/video_favorites/
 ---
 
-## Video Favorites
-<hr />
-### List Video Favorite Objects
-<pre>
-<b>GET</b> https://api.zype.com/consumers/[consumer_id]/video_favorites
-</pre>
+# Video Favorites
 
-#### Macros
+---
+
+## Overview
+The Video Favorites API is used to query, create, and delete Video Favorites for [authenticated Consumers](http://dev.zype.com/api_docs/oauth).
```

Comment:
@imchrissmith i added more descriptions up-front in the overview section. can you take a look?